src/components/Blocking/ReportDomain.js

```diff
@@ -131,7 +132,7 @@ const styles = StyleSheet.create({
     marginBottom: 30,
   },
   btnSkip: {
-    backgroundColor: '#E0E0E0',
+    backgroundColor: COLORS.alto,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hard-coded color values with dynamic ones from a theme object is a good practice as it enhances modularity and consistency.
```diff
- backgroundColor: '#E0E0E0',
+ backgroundColor: COLORS.alto,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->